processor/spanmetricsprocessor/config.go

```diff
@@ -45,4 +45,7 @@ type Config struct {
 	// The dimensions will be fetched from the span's attributes. Examples of some conventionally used attributes:
 	// https://github.com/open-telemetry/opentelemetry-collector/blob/main/model/semconv/opentelemetry.go.
 	Dimensions []Dimension `mapstructure:"dimensions"`
+
+	// MetricKeyToDimensionsCacheSize defines the size of the LRU cache for storing MetricKeyToDimensions cache
```

Comment:
```suggestion
	// MetricKeyToDimensionsCacheSize defines the size of the LRU cache for storing MetricKeyToDimensions cache
	// helping to avoid cache memory growing indefinitely over the lifetime of the collector.
```